packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html

```diff
@@ -18,6 +18,18 @@
             </oui-guide-menu-item>
         </oui-guide-menu>
     </oui-header>
+    <oui-header
+        heading="{{ :: $ctrl.container.name }}"
+        data-ng-if="!$ctrl.archive"
+    ></oui-header>
+    <!--return to conatiner-->
+    <oui-back-button
+        data-ng-if="!$ctrl.archive"
+        aria-label="Aria label for heading"
```

Comment:
**nitpick:** Provide a proper label.

Reply:
removed aria-label property and added data-previous-page property for back button.